Godeps/_workspace/src/github.com/GoogleCloudPlatform/kubernetes/pkg/kubectl/cmd/exec.go

```diff
@@ -176,5 +176,24 @@ func RunExec(f *cmdutil.Factory, cmd *cobra.Command, cmdIn io.Reader, cmdOut, cm
 		SubResource("exec").
 		Param("container", containerName)
 
-	return re.Execute(req, config, args, stdin, cmdOut, cmdErr, tty)
+	postErr := re.Execute(req, config, args, stdin, cmdOut, cmdErr, tty)
+
+	// if we don't have an error, return.  If we did get an error, try a GET because v3.0.0 shipped with exec running as a GET.
+	if postErr == nil {
```

Comment:
> should we scope this to method not supported errors?

It would have to include forbidden too since the old policy would forbid this for most users.  I don't have strong feelings.